app/pb_kits/playbook/pb_bar_graph/bar_graph.rb

```diff
@@ -0,0 +1,109 @@
+module Playbook
+  module PbBarGraph 
+    class BarGraph < Playbook::PbKit::Base
+      PROPS = [
+          :configured_classname,
```

Comment:
Every kit should have these props defined in the PbKit::Base class:
classname, id, data

Please reference other kits